lib/elixir_internal_certificate_web/controllers/api/v1/user_session_controller.ex

```diff
@@ -0,0 +1,37 @@
+defmodule ElixirInternalCertificateWeb.Api.V1.UserSessionController do
+  use ElixirInternalCertificateWeb, :controller
+
+  import Plug.Conn
```

Comment:
might I know why do we need to inport the Plug.Conn here @nvminhtue 🤔

Reply:
Removed in [2a244b9](https://github.com/nvminhtue/elixir_internal_certificate/pull/43/commits/2a244b9899a1ac87b393f781fffbd66bffd7d37e)